src/interface/src/app/treatments/map-base-layer/map-base-layer.component.ts

```diff
@@ -1,22 +1,23 @@
 import { Component } from '@angular/core';
-import { NgForOf } from '@angular/common';
+import { AsyncPipe, NgForOf } from '@angular/common';
 import {
   baseLayerStyles,
   BaseLayerType,
   DEFAULT_BASE_MAP,
 } from '../treatment-map/map-base-layers';
 import { MapConfigState } from '../treatment-map/map-config.state';
+import { of } from 'rxjs';
 
 @Component({
   selector: 'app-map-base-layer',
   standalone: true,
-  imports: [NgForOf],
+  imports: [AsyncPipe, NgForOf],
   templateUrl: './map-base-layer.component.html',
   styleUrl: './map-base-layer.component.scss',
 })
 export class MapBaseLayerComponent {
   baseLayers = Object.keys(baseLayerStyles) as BaseLayerType[];
-  readonly defaultLayer = DEFAULT_BASE_MAP;
+  readonly defaultLayer = this.mapConfigState.baseLayer$ || of(DEFAULT_BASE_MAP);
```

Comment:
this.mapConfigState.baseLayer$ already starts with DEFAULT_BASE_MAP so i dont think you need this OR.
You can just rename this to baseLayer$

Reply:
Ohh perfect! Let me update it